marlowe-playground-client/src/Marlowe/Semantics.purs

```diff
@@ -1201,9 +1186,9 @@ evalValue env state value =
         let
           nn = eval rhs * n
 
-          q = nn `quot` d
+          q = nn `div` d
 
-          r = nn `rem` d
+          r = nn `mod` d
```

Comment:
These are not the same thing, this probably changes the semantics. If there is no quot and rem you can define them like this:

```x quot y = (if ((x < 0) = (y < 0)) then x div y else -(abs x div abs y))
x rem y = x - (x quot y) * y```

Reply:
Ah, I guess it works because Rational normalises and makes at least one positive. Nevermind :)